### DISCUSSION TITLE
Auto Unlock

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8516] [Q&A] [answered]
Hello All,

Is this going to support auto unlock at login ? If yes then for which OS's and what tool would be user ? If not which username and password it use ?

I mean we need to use the same password which we have entered while setting it up.

Also, I am not able to see the Quick Unlock option in Settings, Can someone please direct me ?

Thanks

**Comment by savidudu** [comment]
Hello Team,

Any Update on the above  ?

Thanks

**Comment by savidudu** [comment]
Hello Team,

Any Update on the above ?

Thanks

**Comment by droidmonkey** [comment, marked as answer]
You can only use quick unlock on windows and Mac if you meet the respective requirements of those platforms (windows hello and touch id respectively).

We will not support a native ability to auto unlock at login.

**Reply by savidudu** [reply]
When launching the application then I am not able to see the Quick Unlock option in Settings, Can someone please direct me ? Please help .

Or 

this option is automatically enabled and we just need to enable the option from OS side (windows hello and touch id respectively).

thanks

**Reply by michaelk83** [reply]
https://keepassxc.org/docs/KeePassXC_UserGuide.html#_quick_unlock
